Added uninstall.sh

alanmburr opened the pull request:
Looking through the Issues, I found a few users who wanted an uninstall script. So, I made one!

alanmburr committed:
Added uninstall script and updated the README.md

Update README.md and add section for uninstalling.

alanmburr committed:
Rename the uninstall file

alanmburr committed:
Added [sudo] check

alanmburr committed:
Update uninstall section

alanmburr committed:
Fixed a typo

deepaksp commented:
getting this error after running uninstall script

-bash: /usr/sbin/start-systemd-namespace: No such file or directory

alanmburr committed:
update readme

alanmburr committed:
Update uninstall.sh

alanmburr commented:
Sorry, @deepaksp, `sed` wasn't working. The uninstall script now uses `grep`. Hopefully this works for you.

Sorry, @alanmburr.

alanmburr committed:
Create Makefile

alanmburr committed:
Create LICENSE

alanmburr committed:
Update to match DamionGans

https://github.com/DamionGans/ubuntu-wsl2-systemd-script/blob/master/enter-systemd-namespace

alanmburr committed:
Update to match DamionGans

https://github.com/DamionGans/ubuntu-wsl2-systemd-script/blob/master/start-systemd-namespace

alanmburr committed:
WSL Detection!

alanmburr committed:
Change license from MIT to GPL-3.0

alanmburr committed:
Update README.md

lancer-sudo commented:
omg, thank you very much for this pr.  I met a problem that when I use wsl2(windows 10 vm like?) open the vscode, the path of the vscode terminal is not the path,where I use the command "code .".        
This problem waste my hole day ToT.
Finally I found that this tool will set an env "pwd",  but the value of "pwd" is a path where I  install the wsl(ubuntu), It should actually be the path to "pwd" in the virtual disk of wsl. 
So I'm trying to unstall this tool and find your PR.
Think you very much!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!        I can finally sleep T T😭

PCDSandwichMan commented:
@wackyblackie You are an incredible human being. Thank you so much.

mrconnerton-encorehc commented:
https://devblogs.microsoft.com/commandline/systemd-support-is-now-available-in-wsl/

Gonna uninstall this so I can give the  built in support now released a try

bhatsudo commented:
@wackyblackie Thanks for the script!
I installed the script, but now my WSL2 is not starting and shows the error

`nsenter: failed to parse pid: '25`..

So, now I can't even use the uninstall script. Any idea how I can resolve this? Thanks!

MightyCoderX commented:
@bhatsudo I ran wsl like this 
```cmd
wsl.exe -u root
```

BHellstream commented:
@wackyblackie Thanks for the script it was a life saver
also thanks @MightyCoderX, that also saved me a lot of trouble. 

> ```batchfile
> wsl.exe -u root
> ```

alanmburr committed:
update !

alanmburr committed:
update2 !
